### PR TITLE
fix: Reword summary section

### PR DESCRIPTION
**Pull request by damianopetrungaro**

### CODE REVIEW

**Comment by zeke** [comment]
I agree the opening wording needs improvement and I think this is a good first step.

I also think that we should emphasize the larger point about humans removing themselves from the versioning equation. It's not just about readable commits. It's also about automation. For some projects it's about stepping back and letting bots do your releases for you.

What about some kind of opening tagline that captures this idea? "Write better commit messages and let the robots do the rest." That's not great.. just throwing out ideas here. 🤷‍♂️

**Comment by damianopetrungaro** [comment]
Yeah we can add the `automation` word to describe what you have in mind, give me some time to rephrase it a little bit 😄

**Comment by damianopetrungaro** [comment]
@zeke let me know what do you think about it!
I;ll swash this commit after we decide how to rephrase the sentence 😄

**Comment by damianopetrungaro** [comment]
@hbetts @zeke @bcoe  🏓

File: index.md [hunk 1]
```diff
@@ -7,8 +7,11 @@ redirect_from: /lang/en/
 
 ## Summary
 
-As a software developer, I want to squash feature branches onto `master` and write
-a standardized commit message while doing so.
+As a software developer, I want to create a readable commit history writing standardized commit messages to make easier and explicit what changes have been applied to a project.
```
Comment (hbetts) [inline index.md]: > As a software developer

The first part of the first sentence still feels exclusionary to non Software Developers.

I'm not sure calling out a specific profession adds to the rest of the sentence. I feel the rest of the sentence can stand on its own.

File: index.md [hunk 2]
```diff
@@ -7,8 +7,11 @@ redirect_from: /lang/en/
...
+To accomplish it, the Conventional Commits specification proposes introducing a standardized lightweight
```
Comment (hbetts) [inline index.md]: > proposes introducing a standardized

I'll be honest, I never really understood the phrase `proposes introducing a ...`. From my perspective, the existence of this document means that the standard/convention has already been introduced. No need to _propose_ introducing the standard/convention now.

Maybe: `... the Conventional Commits specification is a lightweight convention on top of ...`

**Comment by zeke** [comment]
Just took another look. I agree with the latest feedback from @hbetts.

**Comment by damianopetrungaro** [comment]
I'll try to complete is asap :busy:

**Comment by bcoe** [comment]
I'm not loving this rewrite compared to the original wording, not going to lie. Could you better explain what your goal is?

**Comment by bcoe** [comment]
as an example, I think this blurb:

```
In software development, it's been my experience that bugs are most often introduced at the boundaries between applications.
Unit testing works great for testing the interactions that a maintainer knows about, but do a poor job of capturing all the interesting, often unexpected, ways that a community puts a library to use.
 Anyone who has upgraded to a new patch version of a dependency, only to watch their
application start throwing a steady stream of 500 errors, knows how important
a readable commit history (and [ideally a well maintained CHANGELOG](http://keepachangelog.com/en/0.3.0/)) is to the ensuing
forensic process.
```

does a good job of advocating to people why they should care about keeping a semantic commit history -- something that sometimes seems a bit pedantic.

tldr; this is a pretty major rewrite of the justification for conventional commits that would require significant changes to every translation, and I'd like to better understand the justification, and to make sure that we're actually replacing it with something better.

**Comment by damianopetrungaro** [comment]
@bcoe See #79  for more context 😄

**Comment by damianopetrungaro** [comment]
@bcoe  We already have a [why use conventional commits](https://conventionalcommits.org/#why-use-conventional-commits) section for this that we may improve if you prefer, but actually, I don't really see the value that this introduction section brings.

> does a good job of advocating to people why they should care about keeping a semantic commit history -- something that sometimes seems a bit pedantic.

**Comment by bcoe** [comment]
@damianopetrungaro if we're removing the introduction, could we perhaps add a link in the drop down along the lines of, "why this exists?" with a note from "Ben Coe"?

I'm comfortable with removing the introduction section.

**Comment by damianopetrungaro** [comment]
@bcoe sure, feel free to write down the sentence, I'll add it to the [why use conventional commits](https://conventionalcommits.org/#why-use-conventional-commits) section.

I'll try to find a place to link this section in the new UI.
Let me know when you can write it down, so if you'll be busy we can merge this one meanwhile 👍

File: index.md (hunk 2 above)
Comment (damianopetrungaro) [reply index.md]: Yeah, it makes sense. 👍

**Comment by hbetts** [comment]
@damianopetrungaro as a path forward, could we split the _Summary_ re-wording from a merger of the _Introduction_ and _Why use Convention Commits_ sections?

**Comment by damianopetrungaro** [comment]
@hbetts  yup, we can do it, but if @bcoe give me that sentence I can add it on the fly and it will save some time 😄.

File: index.md (hunk 1 above)
Comment (bcoe) [reply index.md]: I think the summary should be more to the point about the actual work that someone is undertaking when writing conventional commit messages.

The point of the summary is to tell people what they're actually doing, not to sell them on the value of conventional commit messages -- here's a suggestion:

```When writing software use a standardized commit message, such that the
commit history conveys meaning to developers and can be more easily
consumed by tools.```

File: index.md [hunk 3]
```diff
@@ -64,25 +67,6 @@ see the issue for details on the typos fixed
 
 fixes issue #12
 ```
-
-## Introduction
-
-In software development, it's been my experience that bugs are most often introduced at the boundaries between applications.
```
Comment (bcoe) [inline index.md]: for now let's add this section back and get the new website over the finish line. I'd then be comfortable with talking about moving this section into another part of the website.

I don't want to trim this down to one sentence, I would prefer that we have a full page that speaks to why using conventional commits is an awesome idea -- with the spec itself centering around what this using this convention actual entails.

Comment (damianopetrungaro) [reply index.md]: As I told you before, there's already a section in the website.
We can use that one without adding a new layout that someone needs to write (an internal page have to be different from the homepage).

Comment (bcoe) [reply index.md]: having reflected on this, and looking back at the structure of semver.org, I continue to come back to the argument:

_there's no good reason for us to remove the Introduction section._

I'm happy to work with you and, @zeke , and @hbetts to rework the introductory sentence. And in another pull request I'd be happy to word-smith the content of the Introduction paragraph. 
But, given that the structure of the "Conventional Commits" document is based off the structure of the "Semver" document, and that the two RFCs are closely related, I see a lot of value in keeping close to the same structure to the document.

Removing the Introduction section:

* removes a compelling description of why this specification exists.
* deviates from the structure of the document on semver.org.

tldr; I continue to be a strong -1 on this pull request. But, would love for us to reach a resolution so that we can land the new website design with content we all agree on.

Comment (damianopetrungaro) [reply index.md]: As I told sever times when you read the spec is redundant and doesn't fit the section, we have a specified section to discuss why to use the spec.

Let's move forward, I'll remove it from this PR and discuss in a different issue so we can close this one.

Comment (bcoe) [reply index.md]: sounds like we've agreed to disagree for now 😝

let's get a few third parties discussing this in another issue, since we seem to have trouble agreeing on this one point.

I still think it's worthwhile to rewrite the first sentence though? I'm pretty into just making it jump immediately into the functionality of the spec.

**Comment by damianopetrungaro** [comment]
@zeke @bcoe @hbetts Finally updated.
Please review and let's merge it 💪

**Comment by damianopetrungaro** [comment]
@zeke @bcoe @hbetts 🏓

**Comment by damianopetrungaro** [comment]
@bcoe updated 🚀
